README.MD

```diff
@@ -42,52 +42,49 @@ touch build.yaml
+| REPOSITORIES_HASHICORP_ENTRY | no       | `'https://apt.releases.hashicorp.com $(lsb_release -cs) main'`                     | defined default                                                                                                                  |                                                                                                                                                                        |                                                                                                                                                                                                                                |
+| REPOSITORIES_DOCKER_GPG      | no       | `https://download.docker.com/linux/ubuntu/gpg`                                     | defined default                                                                                                                  |                                                                                                                                                                        |                                                                                                                                                                                                                                |
+| REPOSITORIES_DOCKER__ENTRY   | no       | `'https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable'`             | defined default                                                                                                                  |                                                                                                                                                                        |                                                                                                                                                                                                                                |
+| REPOSITORIES_MICROSOFT_GPG   | no       | `https://packages.microsoft.com/keys/microsoft.asc`                                | defined default                                                                                                                  |                                                                                                                                                                        |                                                                                                                                                                                                                                |
```

Comment:
I don't understand what `defined default` means.

Reply:
`predefined default value` is this better for the understanding ?

Reply:
just `default`.